images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
Yeah, I think we want a centos base image.

I'd like to see the GH repo be `ManageIQ/docker-ruby` and the docker build be `manageiq/ruby` similar to the ones I created for memcached and postgres